hw/ip/otbn/data/otbn.hjson

```diff
@@ -213,16 +213,16 @@
         OTBN has a connection to the EDN allowing it to receive 256-bits of entropy.
       '''
       name: "OTBN.KEYMGR"
-      descr: '''
+      desc:'''
```

Comment:
Nit: do you want to re-insert the whitespace between the colon and the `'`? Know you've replaced `descr: ` with `desc:`. I am not sure if the whitespace is needed though.

Reply:
Good catch, the space is not needed for the tooling, but I added it back for consistency.